src/main/java/nl/novi/techiteasycontroller/Dto/Input/WallBracketInputDto.java

```diff
@@ -0,0 +1,63 @@
+package nl.novi.techiteasycontroller.Dto.Input;
+
+
+import nl.novi.techiteasycontroller.Model.Television;
+
+import java.util.List;
+
+public class WallBracketInputDto {
```

Comment:
Op geen van je inputDto's heb je validatieregels toegepast. (Je moet dan ook in je controllers de @Valid annotatie begruiken)